contracts/dtoken/src/ft.rs

```diff
@@ -47,3 +48,19 @@ impl FungibleTokenReceiver for Contract {
         }
     }
 }
+
+#[near_bindgen]
+impl FungibleTokenResolver for Contract {
```

Comment:
@TymofiiSavinov is it the best way to disable transfer? maybe we should override ft_transfer and ft_transfer_call from FungibleTokenCore?


https://docs.rs/near-contract-standards/latest/near_contract_standards/fungible_token/core/trait.FungibleTokenCore.html

Reply:
Yep, you are right